src/core_modules/capture-core/hooks/useScopeTitleText.js

```diff
@@ -7,7 +8,8 @@ export const useScopeTitleText = (scopeId: ?string) => {
 
     const text = {
         [scopeTypes.EVENT_PROGRAM]: `${programName}`,
-        [scopeTypes.TRACKER_PROGRAM]: `${trackedEntityName} in program: ${programName}`,
+        [scopeTypes.TRACKER_PROGRAM]:
+            `${i18n.t('{{trackedEntityName}} in program', { trackedEntityName })}: ${programName}`,
```

Comment:
For the same reason as mentioned in the other comment I would change this to:

```suggestion
           i18n.t('{{trackedEntityName}} in program: {{programName}}', { trackedEntityName, programName }),
```